src/services/API/Plio.js

```diff
@@ -24,7 +24,14 @@ export default {
 
   getAllPlios() {
     // returns the list of all plios
-    return apiClient().get(process.env.VUE_APP_BACKEND_PLIOS_LIST);
+    return new Promise((resolve, reject) => {
+      apiClient()
+        .get(process.env.VUE_APP_BACKEND_PLIO_NEW)
+        .then((response) => {
+          if (response.status == 200) resolve(response.data);
+          else reject(response);
```

Comment:
TODO - handle the reject section properly

Reply:
why can't we just do the `.get` as before? why do we need to return a promise like this?